Cargo.toml

```diff
@@ -403,6 +403,7 @@ zstd = "0.11.2"
 # for details, see https://github.com/solana-labs/crossbeam/commit/fd279d707025f0e60951e429bf778b4813d1b6bf
 [patch.crates-io]
 crossbeam-epoch = { git = "https://github.com/solana-labs/crossbeam", rev = "fd279d707025f0e60951e429bf778b4813d1b6bf" }
+ntapi = { git = "https://github.com/ryoqun/ntapi", rev = "5980bbab2e0501a8100eb88c12222d664ccb3a0a" }
```

Comment:
needs proper docs off course if we decide to merge this. ref: https://github.com/solana-labs/solana/pull/31893#issuecomment-1575901697

Reply:
done: https://github.com/solana-labs/solana/pull/31961/commits/5280bf0ad9e19de78e622462233c06d06a0b3bd6